index.html

```diff
@@ -8394,6 +8394,10 @@ <h3>Processing actions</h3>
  <li><p>Set the <code>button</code> property of <var>action</var>
   to <var>button</var>.
 
+ <li><p>If <var>action item</var> has subtype property set to
+ "<code>pointerUp</code>", return success with
+ data <a><code>null</code></a>.
```

Comment:
It's still not clear to me. The [`pointerUp` event](https://developer.mozilla.org/en-US/docs/Web/API/Element/pointerup_event) has all these common pointer properties as well. So why shall we return early and not correctly handle all the properties instead?

Reply:
The pointer up event doesn't use any of the below properties for emission.

Reply:
The functional difference is that we don't even validate these properties. So `{type: "pointerUp", button: 0, width: "invalid value"}` wouldn't throw.

But that seems more or less OK to me; it's similar to what we do for other unused properties (e.g. if you provided a `context: []` property we wouldn't notice that it's not valid as a context). However it's a bit of a shift from the original model where all pointer types were treated as more or less interchangable. Probably if we'd started with this model from the start we'd just have ended up with `mouseDown`, `touchDown` and `penDown` rather than `pointerDown`.

From an author's point of view, neither approach helps you catch the error where you get the wrong pointer type and end up sending well-formed (say) pen inputs as a mouse pointer type, with most of the properties being ignored.